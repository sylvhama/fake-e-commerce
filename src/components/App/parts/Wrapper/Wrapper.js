import styled from 'styled-components';

export default styled.section`
  padding-bottom: ${props => props.theme.spacing.reg};
  min-height: calc(100vh - ${props => `${props.theme.size.footer}`});
  color: ${props => props.theme.color.grey};
`;
