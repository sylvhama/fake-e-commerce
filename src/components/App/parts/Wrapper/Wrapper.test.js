import React from 'react';
import 'jest-styled-components';

import { renderWithTheme } from '../../../../tests/helpers';

import Wrapper from './Wrapper';

test('Wrapper renders correctly', () => {
  const rendered = renderWithTheme(<Wrapper>Wrapper</Wrapper>).toJSON();
  expect(rendered).toMatchSnapshot();
  expect(rendered).toHaveStyleRule('min-height', 'calc(100vh - 60px)');
});
