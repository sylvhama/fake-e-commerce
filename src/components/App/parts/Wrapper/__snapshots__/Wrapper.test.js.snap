// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Wrapper renders correctly 1`] = `
.c0 {
  padding-bottom: 16px;
  min-height: calc(100vh - 60px);
  color: #333;
}

<section
  className="c0"
>
  Wrapper
</section>
`;
